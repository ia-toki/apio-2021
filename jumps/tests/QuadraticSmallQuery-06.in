209636825ecd2c38299498cd26890ef71dd128c2
2000 5
1998 1997 1996 1992 1991 1990 1986 1985 1984 1980 1979 1978 1974 1973 1972 1968 1967 1966 1962 1961 1960 1956 1955 1954 1950 1949 1948 1944 1943 1942 1938 1937 1936 1932 1931 1930 1926 1925 1924 1920 1919 1918 1914 1913 1912 1908 1907 1906 1902 1901 1900 1896 1895 1894 1890 1889 1888 1884 1883 1882 1878 1877 1876 1872 1871 1870 1866 1865 1864 1860 1859 1858 1854 1853 1852 1848 1847 1846 1842 1841 1840 1836 1835 1834 1830 1829 1828 1824 1823 1822 1818 1817 1816 1812 1811 1810 1806 1805 1804 1800 1799 1798 1794 1793 1792 1788 1787 1786 1782 1781 1780 1776 1775 1774 1770 1769 1768 1764 1763 1762 1758 1757 1756 1752 1751 1750 1746 1745 1744 1740 1739 1738 1734 1733 1732 1728 1727 1726 1722 1721 1720 1716 1715 1714 1710 1709 1708 1704 1703 1702 1698 1697 1696 1692 1691 1690 1686 1685 1684 1680 1679 1678 1674 1673 1672 1668 1667 1666 1662 1661 1660 1656 1655 1654 1650 1649 1648 1644 1643 1642 1638 1637 1636 1632 1631 1630 1626 1625 1624 1620 1619 1618 1614 1613 1612 1608 1607 1606 1602 1601 1600 1596 1595 1594 1590 1589 1588 1584 1583 1582 1578 1577 1576 1572 1571 1570 1566 1565 1564 1560 1559 1558 1554 1553 1552 1548 1547 1546 1542 1541 1540 1536 1535 1534 1530 1529 1528 1524 1523 1522 1518 1517 1516 1512 1511 1510 1506 1505 1504 1500 1499 1498 1494 1493 1492 1488 1487 1486 1482 1481 1480 1476 1475 1474 1470 1469 1468 1464 1463 1462 1458 1457 1456 1452 1451 1450 1446 1445 1444 1440 1439 1438 1434 1433 1432 1428 1427 1426 1422 1421 1420 1416 1415 1414 1410 1409 1408 1404 1403 1402 1398 1397 1396 1392 1391 1390 1386 1385 1384 1380 1379 1378 1374 1373 1372 1368 1367 1366 1362 1361 1360 1356 1355 1354 1350 1349 1348 1344 1343 1342 1338 1337 1336 1332 1331 1330 1326 1325 1324 1320 1319 1318 1314 1313 1312 1308 1307 1306 1302 1301 1300 1296 1295 1294 1290 1289 1288 1284 1283 1282 1278 1277 1276 1272 1271 1270 1266 1265 1264 1260 1259 1258 1254 1253 1252 1248 1247 1246 1242 1241 1240 1236 1235 1234 1230 1229 1228 1224 1223 1222 1218 1217 1216 1212 1211 1210 1206 1205 1204 1200 1199 1198 1194 1193 1192 1188 1187 1186 1182 1181 1180 1176 1175 1174 1170 1169 1168 1164 1163 1162 1158 1157 1156 1152 1151 1150 1146 1145 1144 1140 1139 1138 1134 1133 1132 1128 1127 1126 1122 1121 1120 1116 1115 1114 1110 1109 1108 1104 1103 1102 1098 1097 1096 1092 1091 1090 1086 1085 1084 1080 1079 1078 1074 1073 1072 1068 1067 1066 1062 1061 1060 1056 1055 1054 1050 1049 1048 1044 1043 1042 1038 1037 1036 1032 1031 1030 1026 1025 1024 1020 1019 1018 1014 1013 1012 1008 1007 1006 1002 1001 1000 996 995 994 990 989 988 984 983 982 978 977 976 972 971 970 966 965 964 960 959 958 954 953 952 948 947 946 942 941 940 936 935 934 930 929 928 924 923 922 918 917 916 912 911 910 906 905 904 900 899 898 894 893 892 888 887 886 882 881 880 876 875 874 870 869 868 864 863 862 858 857 856 852 851 850 846 845 844 840 839 838 834 833 832 828 827 826 822 821 820 816 815 814 810 809 808 804 803 802 798 797 796 792 791 790 786 785 784 780 779 778 774 773 772 768 767 766 762 761 760 756 755 754 750 749 748 744 743 742 738 737 736 732 731 730 726 725 724 720 719 718 714 713 712 708 707 706 702 701 700 696 695 694 690 689 688 684 683 682 678 677 676 672 671 670 666 665 664 660 659 658 654 653 652 648 647 646 642 641 640 636 635 634 630 629 628 624 623 622 618 617 616 612 611 610 606 605 604 600 599 598 594 593 592 588 587 586 582 581 580 576 575 574 570 569 568 564 563 562 558 557 556 552 551 550 546 545 544 540 539 538 534 533 532 528 527 526 522 521 520 516 515 514 510 509 508 504 503 502 498 497 496 492 491 490 486 485 484 480 479 478 474 473 472 468 467 466 462 461 460 456 455 454 450 449 448 444 443 442 438 437 436 432 431 430 426 425 424 420 419 418 414 413 412 408 407 406 402 401 400 396 395 394 390 389 388 384 383 382 378 377 376 372 371 370 366 365 364 360 359 358 354 353 352 348 347 346 342 341 340 336 335 334 330 329 328 324 323 322 318 317 316 312 311 310 306 305 304 300 299 298 294 293 292 288 287 286 282 281 280 276 275 274 270 269 268 264 263 262 258 257 256 252 251 250 246 245 244 240 239 238 234 233 232 228 227 226 222 221 220 216 215 214 210 209 208 204 203 202 198 197 196 192 191 190 186 185 184 180 179 178 174 173 172 168 167 166 162 161 160 156 155 154 150 149 148 144 143 142 138 137 136 132 131 130 126 125 124 120 119 118 114 113 112 108 107 106 102 101 100 96 95 94 90 89 88 84 83 82 78 77 76 72 71 70 66 65 64 60 59 58 54 53 52 48 47 46 42 41 40 36 35 34 30 29 28 24 23 22 18 17 16 12 11 10 6 5 4 1 2 3 7 8 9 13 14 15 19 20 21 25 26 27 31 32 33 37 38 39 43 44 45 49 50 51 55 56 57 61 62 63 67 68 69 73 74 75 79 80 81 85 86 87 91 92 93 97 98 99 103 104 105 109 110 111 115 116 117 121 122 123 127 128 129 133 134 135 139 140 141 145 146 147 151 152 153 157 158 159 163 164 165 169 170 171 175 176 177 181 182 183 187 188 189 193 194 195 199 200 201 205 206 207 211 212 213 217 218 219 223 224 225 229 230 231 235 236 237 241 242 243 247 248 249 253 254 255 259 260 261 265 266 267 271 272 273 277 278 279 283 284 285 289 290 291 295 296 297 301 302 303 307 308 309 313 314 315 319 320 321 325 326 327 331 332 333 337 338 339 343 344 345 349 350 351 355 356 357 361 362 363 367 368 369 373 374 375 379 380 381 385 386 387 391 392 393 397 398 399 403 404 405 409 410 411 415 416 417 421 422 423 427 428 429 433 434 435 439 440 441 445 446 447 451 452 453 457 458 459 463 464 465 469 470 471 475 476 477 481 482 483 487 488 489 493 494 495 499 500 501 505 506 507 511 512 513 517 518 519 523 524 525 529 530 531 535 536 537 541 542 543 547 548 549 553 554 555 559 560 561 565 566 567 571 572 573 577 578 579 583 584 585 589 590 591 595 596 597 601 602 603 607 608 609 613 614 615 619 620 621 625 626 627 631 632 633 637 638 639 643 644 645 649 650 651 655 656 657 661 662 663 667 668 669 673 674 675 679 680 681 685 686 687 691 692 693 697 698 699 703 704 705 709 710 711 715 716 717 721 722 723 727 728 729 733 734 735 739 740 741 745 746 747 751 752 753 757 758 759 763 764 765 769 770 771 775 776 777 781 782 783 787 788 789 793 794 795 799 800 801 805 806 807 811 812 813 817 818 819 823 824 825 829 830 831 835 836 837 841 842 843 847 848 849 853 854 855 859 860 861 865 866 867 871 872 873 877 878 879 883 884 885 889 890 891 895 896 897 901 902 903 907 908 909 913 914 915 919 920 921 925 926 927 931 932 933 937 938 939 943 944 945 949 950 951 955 956 957 961 962 963 967 968 969 973 974 975 979 980 981 985 986 987 991 992 993 997 998 999 1003 1004 1005 1009 1010 1011 1015 1016 1017 1021 1022 1023 1027 1028 1029 1033 1034 1035 1039 1040 1041 1045 1046 1047 1051 1052 1053 1057 1058 1059 1063 1064 1065 1069 1070 1071 1075 1076 1077 1081 1082 1083 1087 1088 1089 1093 1094 1095 1099 1100 1101 1105 1106 1107 1111 1112 1113 1117 1118 1119 1123 1124 1125 1129 1130 1131 1135 1136 1137 1141 1142 1143 1147 1148 1149 1153 1154 1155 1159 1160 1161 1165 1166 1167 1171 1172 1173 1177 1178 1179 1183 1184 1185 1189 1190 1191 1195 1196 1197 1201 1202 1203 1207 1208 1209 1213 1214 1215 1219 1220 1221 1225 1226 1227 1231 1232 1233 1237 1238 1239 1243 1244 1245 1249 1250 1251 1255 1256 1257 1261 1262 1263 1267 1268 1269 1273 1274 1275 1279 1280 1281 1285 1286 1287 1291 1292 1293 1297 1298 1299 1303 1304 1305 1309 1310 1311 1315 1316 1317 1321 1322 1323 1327 1328 1329 1333 1334 1335 1339 1340 1341 1345 1346 1347 1351 1352 1353 1357 1358 1359 1363 1364 1365 1369 1370 1371 1375 1376 1377 1381 1382 1383 1387 1388 1389 1393 1394 1395 1399 1400 1401 1405 1406 1407 1411 1412 1413 1417 1418 1419 1423 1424 1425 1429 1430 1431 1435 1436 1437 1441 1442 1443 1447 1448 1449 1453 1454 1455 1459 1460 1461 1465 1466 1467 1471 1472 1473 1477 1478 1479 1483 1484 1485 1489 1490 1491 1495 1496 1497 1501 1502 1503 1507 1508 1509 1513 1514 1515 1519 1520 1521 1525 1526 1527 1531 1532 1533 1537 1538 1539 1543 1544 1545 1549 1550 1551 1555 1556 1557 1561 1562 1563 1567 1568 1569 1573 1574 1575 1579 1580 1581 1585 1586 1587 1591 1592 1593 1597 1598 1599 1603 1604 1605 1609 1610 1611 1615 1616 1617 1621 1622 1623 1627 1628 1629 1633 1634 1635 1639 1640 1641 1645 1646 1647 1651 1652 1653 1657 1658 1659 1663 1664 1665 1669 1670 1671 1675 1676 1677 1681 1682 1683 1687 1688 1689 1693 1694 1695 1699 1700 1701 1705 1706 1707 1711 1712 1713 1717 1718 1719 1723 1724 1725 1729 1730 1731 1735 1736 1737 1741 1742 1743 1747 1748 1749 1753 1754 1755 1759 1760 1761 1765 1766 1767 1771 1772 1773 1777 1778 1779 1783 1784 1785 1789 1790 1791 1795 1796 1797 1801 1802 1803 1807 1808 1809 1813 1814 1815 1819 1820 1821 1825 1826 1827 1831 1832 1833 1837 1838 1839 1843 1844 1845 1849 1850 1851 1855 1856 1857 1861 1862 1863 1867 1868 1869 1873 1874 1875 1879 1880 1881 1885 1886 1887 1891 1892 1893 1897 1898 1899 1903 1904 1905 1909 1910 1911 1915 1916 1917 1921 1922 1923 1927 1928 1929 1933 1934 1935 1939 1940 1941 1945 1946 1947 1951 1952 1953 1957 1958 1959 1963 1964 1965 1969 1970 1971 1975 1976 1977 1981 1982 1983 1987 1988 1989 1993 1994 1995 1999 2000
999 999 1999 1999
190 737 1402 1932
165 1345 1590 1659
1199 1803 1824 1837
1045 1140 1623 1647
