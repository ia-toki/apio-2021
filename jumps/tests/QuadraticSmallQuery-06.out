fc5005cd102fffc504bc1615010ebf88823d65f3
OK
667
1
1
14
322
